### PR TITLE
Exam Terms & Conditions

### PR DESCRIPTION
**Pull request by tarakenishauy**
Incomplete and format is not quite right but placed placeholders for the client demo

### CODE REVIEW

File: client/src/css/Exam.css [hunk 1]
```diff
@@ -18,15 +18,15 @@
...
+    background-color: #109cfc;
     border-radius: 6px;
     color: white;
     padding: 20px;
 }
 
 .preview {
     height: 550px;
-    background-color: #2b2d42;
+    background-color: #109cfc;
```
Comment (abrarbarbhuyia) [inline client/src/css/Exam.css:29]: Ah just changed these to #2b2d42 for the demo, can you edit it bacK?

File: client/src/views/ExamStart.js [hunk 2]
```diff
@@ -91,7 +91,7 @@ function ExamStart() {
                       component="th"
                       scope="row"
                       sx={{
-                        backgroundColor: "#2b2d42",
+                        backgroundColor: "#109cfc",
```
Comment (abrarbarbhuyia) [inline client/src/views/ExamStart.js:94]: Same as other comment

File: client/src/views/ExamStart.js [hunk 3]
```diff
@@ -148,7 +175,7 @@ function ExamStart() {
             variant="contained"
             className="continueButton"
             style={{
-              backgroundColor: isChecked ? "#2b2d42" : "grey",
+              backgroundColor: isChecked ? "#109cfc" : "grey",
```
Comment (abrarbarbhuyia) [inline client/src/views/ExamStart.js:178]: Same as other comment

**Review by abrarbarbhuyia — APPROVED** [review]
Looks good! Added some comments abt the color stuff for exam start/session.

**Review by gbashour — COMMENTED** [review]
- I think it'll be safer to remove any changes to package.json as @ProgKorn  has resolved dependency issues (unless there are newly added packages)
- Make sure it's updated with the latest changes for ObjectRecognition etc

**Pull request closed without merging**